src/App.tsx

```diff
@@ -1,24 +1,135 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { addTodo, deleteTodo, getTodos } from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { FilterStatus } from './types/FilterStatus';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { getFilteredTodos } from './utils/filterTodos';
+import { TodoHeader } from './components/TodoHeader/TodoHeder';
+import { ErrorNotification } from
+  './components/ErrorNotification/ErrorNotification';
 
-const USER_ID = 0;
+const USER_ID = 11509;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMesssage, setErrorMesssage] = useState('');
+  const [selectedFilter, setSelectedFilter]
+  = useState<FilterStatus>(FilterStatus.All);
+
+  const [title, setTitle] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isDisabled, setIsDisabled] = useState(false);
+
+  useEffect(() => {
+    setErrorMesssage('');
+
+    getTodos(USER_ID)
+      .then(setTodos)
+      .catch(() => {
+        setErrorMesssage('Unable to load todos');
+
+        setTimeout(() => {
+          setErrorMesssage('');
+        }, 3000);
```

Comment:
You can trigger timeout in the `ErrorNotification` component in case of any error